themes/default/content/blog/introducing-dev-releases/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Introducing Dev Releases"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-03-27T17:20:44+01:00
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc:
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - thomas-gummerer
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+  - cli
+  - sdk
+  - pre-release
+---
+
+At Pulumi, the stability of our releases is critically important.  A lot of infrastructure is built and deployed using Pulumi, and bugs in the way that infrastructure is deployed can cause long outages.  While we put a lot of emphasis on unit and integration testing, we wanted an additional layer to make sure we always deliver stable releases.  To that end we introduced dev releases.  The following blog post will talk more about how we are using them internally, and how you can too.
+
+<!--more-->
+
+## What are dev releases?
+
+On every push to the main branch, in other words every time a pull request is merged, we publish a new dev release for both the CLI and the pulumi core SDKs.  For the CLI, we publish the binaries for all supported platforms to S3, while for the SDKs we create pre-release versions in whatever form the various package managers support.
+
+This means these dev releases always include all the latest features and bug fixes, that are going to go into a stable release a bit later.  And while it was always possible to get the latest CLI or SDKs with some moderate effort depending on the platform and language, having these be proper releases makes for a much more standardized experience on how to install them.
```

Comment:
Nit: "going to go into" feels awkward, and "a bit later" seems vague

maybe "that will be in the next stable release" rather than "that are going to go into a stable release a bit later" -- or if we don't want to commit to saying "next stable release" you could say "that will be in a a future stable release"